src/app/introduction/introduction.component.scss

```diff
@@ -0,0 +1,162 @@
+@import "../styles/components/_variables";
```

Comment:
so, which of these 2 `introduction.component.scss` is correct?
you are only going to cause merge conflicts at this rate